src/app/task/taskStatus.interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface TaskStatusInterface {
+  done?: string,
+  toDo: string,
+  inProgress?: string
+}
```

Comment:
totally wrong - status is represented as a string but not as an object. so:
1) if you define possible values as string literals
export const typesT = {
  one: 'one',
  two: 'two',
  three: 'three'
};

export type TaskStatusType = keyof typeof typesT;

2) or if you define constants as enum:
export enum typesT {
  one = 'one';
  two = 'two';
  three = 'three';
};

export type TaskStatusType = typesT;